plugins/lsp/language-servers/pylsp.nix

```diff
@@ -539,7 +539,7 @@ in
               (optional (isEnabled plugins.${pluginName}) (
                 nixPackage.overridePythonAttrs (old: {
                   # Get rid of the python-lsp-server dependency
-                  propagatedBuildInputs = filter (dep: dep.pname != "python-lsp-server") old.propagatedBuildInputs;
+                  dependencies = filter (dep: dep.pname != "python-lsp-server") old.dependencies;
```

Comment:
Is rather go with https://github.com/nix-community/nixvim/pull/1893 since that'll solve the issue gracefully for users on both new and old channel locks.

I think we can merge that PR and then rebase this branch on top of it?